src/core/common/ocispec/ocispec.hpp

```diff
@@ -56,13 +65,22 @@ class OCISpecItf {
     virtual Error SaveImageManifest(const String& path, const ImageManifest& manifest) = 0;
 
     /**
-     * Loads OCI image spec.
+     * Loads OCI image spec from file.
      *
      * @param path file path.
      * @param imageSpec image spec.
      * @return Error.
      */
-    virtual Error LoadImageSpec(const String& path, ImageSpec& imageSpec) = 0;
+    virtual Error ImageSpecFromFile(const String& path, ImageSpec& imageSpec) = 0;
```

Comment:
Add corresponding implementation for aos_core_cpp

Reply:
will do.